enterprise/backend/src/metabase_enterprise/semantic_search/task/indexer.clj

```diff
@@ -32,14 +31,15 @@
  SemanticSearchIndexer []
   org.quartz.Job
   (execute [_ _]
-    (log/with-context {:quartz-job-type 'SemanticSearchIndexer}
-      (when (search.engine/supported-engine? :search.engine/semantic)
-        (try
-          (vreset! execution-thread-ref (Thread/currentThread))
-          (semantic-search.indexer/quartz-job-run! (semantic.env/get-pgvector-datasource!) (semantic.env/get-index-metadata))
-          (finally
-            (locking execution-thread-ref
-              (vreset! execution-thread-ref nil)))))))
+    (when (semantic.u/semantic-search-available?)
+      (log/with-context {:quartz-job-type 'SemanticSearchIndexer}
+        (when (search.engine/supported-engine? :search.engine/semantic)
+          (try
+            (vreset! execution-thread-ref (Thread/currentThread))
+            (semantic-search.indexer/quartz-job-run! (semantic.env/get-pgvector-datasource!) (semantic.env/get-index-metadata))
+            (finally
+              (locking execution-thread-ref
+                (vreset! execution-thread-ref nil))))))))
```

Comment:
nit: indent looks wrong

Reply:
where? `cljfmt` would fail if it was wrong